integration/dataproc/build.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+#
+# The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+# (the "License"). You may not use this work except in compliance with the License, which is
+# available at www.apache.org/licenses/LICENSE-2.0
+#
+# This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+# either express or implied, as more fully set forth in the License.
+#
+# See the NOTICE file distributed with this work for information regarding copyright ownership.
+#
+
+# Running this script will create edited versions of the files in the .generated directory
+set -e
+
+ALLUXIO_DOWNLOAD_URL=${1}
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+mkdir -p ${DIR}/.generated
+cp ${DIR}/alluxio.sh ${DIR}/.generated
+
+# replace ALLUXIO_DOWNLOAD_URL in dataproc init script (alluxio.sh)
+if [[ -n ${ALLUXIO_DOWNLOAD_URL} ]]; then
+    if [[ -n $(grep "readonly ALLUXIO_DOWNLOAD_URL=\"${ALLUXIO_DOWNLOAD_URL}\"" "${DIR}/alluxio.sh") ]]; then
+    echo "ERROR: ${ALLUXIO_DOWNLOAD_URL} is already set. No change was made to alluxio-emr.sh"
```

Comment:
alluxio-emr.sh -> alluxio.sh
probably more convenient to make this a bash variable

Reply:
it'll be difficult to use `build.sh` generically if i need to know the file names beforehand

Reply:
i meant as a bash variable like `local FILENAME=alluxio.sh`, not an argument into the script